sklearn/svm/classes.py

```diff
@@ -405,7 +405,8 @@ class SVC(BaseSVC):
          It must be one of 'linear', 'poly', 'rbf', 'sigmoid', 'precomputed' or
          a callable.
          If none is given, 'rbf' will be used. If a callable is given it is
-         used to precompute the kernel matrix.
+         used to precompute the kernel matrix from data matrices. The
+         kernel matrix should be an array of shape ``(n_samples, n_features)``
```

Comment:
It should be `(n_samples, n_samples)`. Also "The kernel matrix" is ambiguous; I suggest

    used to precompute the kernel matrix from data matrices; that matrix
    should be an array of shape ``(n_samples, n_samples)``.